dev/docker-compose.yml

```diff
@@ -127,13 +129,14 @@ services:
       - DB_VENDOR=H2
       - KEYCLOAK_CLIENT_ID=conjurClient
       - KEYCLOAK_REDIRECT_URI=http://locallhost.com/
-      - KEYCLOAK_CLIENT_SECRET=1234
+      - KEYCLOAK_CLIENT_SECRET=d7047915-9029-45b8-9bd6-7ec5c2f75e5b
```

Comment:
Can we add a comment explaining where this magic UUID comes from?

Reply:
i don't really remember what this is and if it has any logic. @nessiLahav doesn't remember as well. I am fine with leaving it as it is without investigating this now.

Reply:
Maybe I'm misunderstanding, but I'm confused... it's marked as a change added for this commit.  Why would we merge a code change that no one knows the purpose of?   What happens if we change it back to `1234` or some other value -- does it stop working?

Reply:
you previously made some changes to `ci/docker-compose` around the OIDC containers and forgot to change also `de/docker-compose` so the dev env of OIDC broke. As part of this fix i just took the client id from the `ci/docker-compose`. 

However, i tested it now with `1234` and it works so i'll change it to that in both places. better to have `1234` than a random UUID. it is clearer that it's just some sequence.